build.sbt

```diff
@@ -24,34 +23,32 @@ lazy val root = (project in file("."))
      )
   )
 
-scalaVersion := "2.12.16"
+scalaVersion := "2.13.0"
 
 val awsVersion = "1.12.129"
 
+// Until all dependencies are on scala-java8-compat v1.x, this avoids unnecessary fatal eviction errors
+ThisBuild / libraryDependencySchemes += "org.scala-lang.modules" %% "scala-java8-compat" % VersionScheme.Always
```

Comment:
Does this need to be here and in plugins.sbt?

Reply:
Looks like we don't need this or `libraryDependencySchemes` line in plugins.sbt - it was needed to compile after upgrading play to  2.9, but works locally without them on play 3.0 - will check on CODE again once built